addons/l10n_ug/data/template/account.account-ug.csv

```diff
@@ -0,0 +1,259 @@
+id,name,code,account_type,tag_ids,reconcile
+141,Property Income,141,income_other,,False
+142,Sale of goods and services,142,income,,False
+142111,Rent & rates - produced assets,142111,income,,False
```

Comment:
So sale of goods go in to 142000 and rent & rates in 142111?  (sure it is not more of a name for all accounts under 142?)  Maybe 1420 instead of 142 can avoid the confusion.